src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2039,6 +2116,9 @@
 helps['network watcher'] = """
     type: group
     short-summary: Manage the Azure Network Watcher.
+    long-summary: >
+        Network Watcher assists with monitoring and diagnosing conditions at a network scenario level. 
+            Visit https://docs.microsoft.com/en-us/azure/network-watcher/
```

Comment:
Missing key words for the `Visit` link. Should read similar to other links - `For more information visit`, `To learn more about Network Watcher see`, etc.

Reply:
Fixed in my branch, but it seems there are two patterns "For more information" and "To learn more".  If you have a preference I can update them all for consistency.

Reply:
I think "For more information" is the preferable of the two, but that could change depending on time of day. Consistency is the most important thing though, so let's go for that.